apps/api/src/fcm/fcm.service.ts

```
@@ -36,21 +36,67 @@ export class FcmService {
+    } else {
+      //TODO 이 부분은 알림을 허용한 user에게만 알림이 가도록 수정해야 함.
+      //(getAllFcmTokens 함수를 다른 함수로 대체해야 한다.)
+      totalTokens = (await this.fcmRepository.getAllFcmTokens()).map(
+        ({ fcmTokenId }) => fcmTokenId,
+      );
+    }
+
+    const batches = totalTokens.reduce((acc, token, index) => {
+      if (index % 100 === 0) return [[token], ...acc];
+      const [first, ...array] = acc;
+      return [[...first, token], ...array];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)